openlibrary/plugins/openlibrary/js/carousels.js

```diff
@@ -59,6 +61,47 @@ const Carousel = {
             slidesToScroll: a,
             responsive: responsive_settings
         });
+
+        var addWork = function(work) {
```

Comment:
@tabshaikh the availability info still needs to be updated to use the actual API data rather than just always showing borrow